### PR TITLE
cleaning up methods moved from base test to cluster

### PR DESCRIPTION
**Pull request by nshidlin**
/cc @tsorya @lalon4

### CODE REVIEW

File: discovery-infra/tests/base_test.py [hunk 1]
```diff
@@ -46,24 +46,12 @@ def get_cluster_func(cluster_name: Optional[str] = None):
 
         for cluster in clusters:
             logging.info(f'--- TEARDOWN --- deleting created cluster {cluster.id}')
```
Comment (lalon4) [inline discovery-infra/tests/base_test.py:48]: can you please add a separate line log for TEARDOWN?
that way it is easy to see in logs when test ended. it separate the screen

File: discovery-infra/test_infra/helper_classes/cluster.py [hunk 2]
```diff
@@ -299,7 +307,7 @@ def download_installation_logs(self, path):
         self.api_client.download_cluster_logs(self.id, path)
 
     def get_install_config(self):
-        return self.api_client.get_cluster_install_config(self.id)
+        return yaml.load(self.api_client.get_cluster_install_config(self.id), Loader=yaml.SafeLoader)
```
Comment (tsorya) [inline discovery-infra/test_infra/helper_classes/cluster.py:310]: not sure i understand why we need it?

File: discovery-infra/tests/base_test.py (hunk 1 above)
Comment (tsorya) [reply discovery-infra/tests/base_test.py:48]: it is relevant not only to cluster

File: discovery-infra/test_infra/helper_classes/cluster.py (hunk 2 above)
Comment (nshidlin) [reply discovery-infra/test_infra/helper_classes/cluster.py:310]: I see if proxy values are set in the install config

File: discovery-infra/tests/base_test.py (hunk 1 above)
Comment (nshidlin) [reply discovery-infra/tests/base_test.py:48]: yeah, i can add TEARDOWN log to the other fixtures
wdty?

Comment (nshidlin) [reply discovery-infra/tests/base_test.py]: done

File: discovery-infra/test_infra/helper_classes/cluster.py (hunk 2 above)
Comment (tsorya) [reply discovery-infra/test_infra/helper_classes/cluster.py:310]: but you get json? why u need to change it to yaml?

Comment (tsorya) [reply discovery-infra/test_infra/helper_classes/cluster.py:310]: got it

**Pull request merged**